ui/slider.js

```diff
@@ -210,9 +202,8 @@ return $.widget( "ui.slider", $.ui.mouse, {
 
 		this._handleIndex = index;
 
-		closestHandle
-			.addClass( "ui-state-active" )
-			.focus();
+		closestHandle.focus();
+		this._addClass( closestHandle, null, "ui-state-active" );
```

Comment:
Should add the class first, then call `.focus()`. More consistent with the previous behaviour and makes sense in general.